tests/unittests/OperatorGradTest.cpp

```diff
@@ -105,6 +105,57 @@ TEST_P(OperatorGradTest, concat) {
   EXPECT_TRUE(expectedRight.isEqual(*getGradTensor(varGrads, B)));
 }
 
+TEST_P(OperatorGradTest, conv) {
+  ENABLED_BACKENDS(Interpreter, CPU);
+
+  auto *X = mod_.createPlaceholder(
+      ElemKind::FloatTy, {1, 3 /* width */, 3 /* height */, 1 /* depth */}, "x",
+      false);
+  bindings_.allocate(X)->getHandle() = {0, 1, 2, 3, 4, 5, 6, 7, 8};
+
+  ConvolutionNode *O =
+      F_->createConv(bindings_, "conv", X, 1 /* outChannels */, 2 /* kernel */,
+                     1 /* stride */, 0 /* pad */, 1 /* group */);
+
+  // Note that filter is trainable, but probably it's okay since we run the
+  // graph only once.
```

Comment:
This can be:  llvm::cast<Placeholder> -> cast<Placeholder>.

Add using llvm::cast at the top of the file.